definition/definition.go

```diff
@@ -31,14 +31,43 @@ type Chain interface {
 // carry on, call Chain.Continue() and pass the context.
 type Middleware func(context.Context, Chain) error
 
+// Operator is used to operate an object and return an object as replacement.
+//
+// For example:
+//  A converter:
+//    type ConverterForAnObject struct{}
+//    func (c *ConverterForAnObject) Kind() {return "converter"}
```

Comment:
Is this `Kind` pre-defined?

Reply:
to answer myself, the answer seems to be yes

Reply:
I have considered about it. But I think pre-defined `Kind` is useless.  We don't limit the usage of operators and `Kind` is customized. Or we just define an `OperatorKind` for it.